package.json

```diff
@@ -50,5 +50,6 @@
       "mocha": true
     },
     "extends": "eslint:recommended"
-  }
+  },
+  "type": "./types/index.d.ts"
```

Comment:
Move this line right under the "main" where it belongs. See other JS/TS repos for inspiration.